docs/.custom_wordlist.txt

```diff
@@ -6,6 +6,7 @@ Github
 frictionless
 COU
 cou
+colocated
```

Comment:
This is a duplicated entry (see line 46)

When we do documentation review, I think it would be a good idea to order this list alphabetically to avoid duplication.

Reply:
I agree...it's easy to add duplicates if we don't have them alphabetical ordered